include/boost/geometry/io/dsv/write.hpp

```diff
@@ -27,6 +27,7 @@
 #include <boost/range/begin.hpp>
 #include <boost/range/end.hpp>
 #include <boost/range/value_type.hpp>
+#include <boost/range/size.hpp>
```

Comment:
After updating Boost to the most recent develop on my notebook (because of swap), this missing include was problematic for some unit tests, which is the reason why I checked all usages.

Reply:
indeed, this is caught by github actions too.